jdk/README.md

```diff
@@ -0,0 +1,40 @@
+ansible-role-jdk
+=================
+
+An ansible role for installing openjdk.
+
+
+Requirements
+------------
+
+The target host should be setup prior to the run. This role has been setup and has been tested against Centos8 for use of AMIs. The requirements include:
+- CentOS 8 based AMI
+- Python3 installed
+
+Role Variables
+--------------
+
+Variable         |Default     | Description
+-----------------|:------:    |-------------
+version          |`1.8.0`     | The java version to install*
```

Comment:
Can the asterisk above be removed?

Reply:
Updated to remove asterisk and some other minor changes to the readme :)